packages/babel-generator/src/generators/methods.ts

```diff
@@ -11,7 +11,11 @@ export function _params(
   this._parameters(node.params, node);
   this.token(")");
 
-  this.print(node.returnType, node);
+  if (node.returnType) {
```

Comment:
Can we set `_noLineTerminator` only when `node` is an `ArrowFunctionExpression`?